app/Http/Controllers/MatchdayController.php

```diff
@@ -19,71 +20,84 @@ public function index()
     }
 
     /**
-     * Show the form for editing the specified resource.
+     * Show the form for creating a new resource.
      *
-     * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function edit($id)
+    public function create()
     {
-        // TODO show an EXISTING matchday (ID!) in a form for editing (route '/termine/bearbeiten'; view 'matchdays.edit'); followed by update
+        // TODO show a form for creating a NEW matchday (route '/termine/erstellen'; view 'matchdays.create'); followed by store
+        return view('matchdays/create');
     }
 
     /**
-     * Update the specified resource in storage.
+     * Store a newly created resource in storage.
      *
      * @param  \Illuminate\Http\Request  $request
-     * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, $id)
+    public function store(Request $request)
     {
         // TODO validate the request and 
-        // TODO update an EXISTING matchday (ID!) in storage (no own route; after view 'matchdays.edit')
+        // TODO store a NEW matchday in storage (no own route; after view 'matchdays.create')
     }
 
     /**
-     * Remove the specified resource from storage.
+     * Display the specified resource.
      *
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function destroy($id)
+    public function show(Matchday $matchdays)
     {
-        // TODO delete an EXISTING matchday (ID!) (no own route; on view 'matchdays.edit')
+        // show a SPECIFIC matchday (no route; no view); 'anzeigen' leads to 'planner'
+        $players = \App\Player::all();
+
+        return view('planner', [
+            'matchdays' => $matchdays,
```

Comment:
Why is this variable called $matchdays (plural) while it's actually just for one matchday?